tests/workflows/components/workflows.libsonnet

```diff
@@ -138,7 +138,8 @@
         },
         spec: {
           entrypoint: "e2e",
-          ttlSecondsAfterFinished: 3600,
+          ttlSecondsAfterFinished: 8*60*60,
```

Comment:
I don't think we should be investing anymore in ksonnet/jsonnet

Reply:
Agree, I should move to Tekton or python-based Argo workflow